src/poetry/core/packages/package.py

```diff
@@ -69,6 +70,13 @@ def __init__(
         """
         from poetry.core.version.markers import AnyMarker
 
+        if pretty_version is not None:
+            warnings.warn(
+                "The `pretty_version` parameter is deprecated and will be removed"
+                " in a future release.",
+                DeprecationWarning,
+            )
```

Comment:
```suggestion
                DeprecationWarning,
                stacklevel=2,
            )
```

It seems like stacklevel is missing in the other deprecations in this file, too.